.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,13 @@
+---
+name: Bug report
+about: Create a report to help us improve
+---
+
+<!-- Please search existing issues to avoid creating duplicates. -->
+
+- Version:
+
+Steps to Reproduce:
+
+1.
+2.
```

Comment:
OS details, package version etc is also needed here

Reply:
Version is already asked, but I will add OS details.